src/client/components/Trip/Trip.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUserAlt } from '@fortawesome/free-solid-svg-icons';
 
 import TripButton from './TripButton';
 import PATHS from '../../constants/paths';
 
 const Trip = props => {
-  let { trip, userId } = props;
+  let { trip, userId, showModal } = props;
   let { trip_id, destination, start_date, end_date, members } = trip;
```

Comment:
use `const` for both

only use `let` if the variable is changing types. e.g. from `null` to `object` or smth.